build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -274,7 +275,7 @@ public static BuildTargetResult PackagePkgProjects(BuildTargetContext c)
                     .Environment("__WorkaroundCliCoreHostVer", hostVersion.LatestHostVersion.WithoutSuffix)
                     .Environment("__WorkaroundCliCoreHostBuildMajor", hostVersion.LatestHostBuildMajor)
                     .Environment("__WorkaroundCliCoreHostBuildMinor", hostVersion.LatestHostBuildMinor)
-                    .Environment("__WorkaroundCliCoreHostVersionTag", hostVersion.LatestHostPrerelease)
+                    .Environment("__WorkaroundCliCoreHostVersionTag", buildVersion.ReleaseSuffix)
```

Comment:
We are using this for `Microsoft.NETCore.App`, so I killed the `LatestHostPrerelease` which is a dup of `ReleaseSuffix` and just used the `ReleaseSuffix`. This is consistent with respect to all the other places in the repo for this purpose.